.editorconfig

```diff
@@ -0,0 +1,14 @@
+# This file is for unifying the coding style for different editors and IDEs
+# editorconfig.org
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = false
+insert_final_newline = true
+indent_style = tab
```

Comment:
indent_style conflicts with the airbrb styleguide which you have for eslint, please change to spaces.
https://github.com/airbnb/javascript#whitespace

Reply:
Updated to be indent_style=space to match the airbnb styles